From: "Igor B. Poretsky" <poretsky@mlbox.ru>
Date: Sat, 31 May 2025 12:03:29 +0300
Subject: Removed package-initialize call

Being executed multiple times it causes warning window popup. And it
seems to be not necessary here.
---
 lisp/emacspeak-preamble.el | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/lisp/emacspeak-preamble.el b/lisp/emacspeak-preamble.el
index 488fe5d..2fc13ec 100644
--- a/lisp/emacspeak-preamble.el
+++ b/lisp/emacspeak-preamble.el
@@ -52,10 +52,6 @@
 (require 'emacspeak-speak)
 (require 'emacspeak-sounds)
 (require 'emacspeak-keymap)
-(eval-when-compile
-  (condition-case nil
-      (package-initialize)
-    (error "Check your elpa packages --- something is broken")))
 
 ;;}}}
 
